src/js/components/AppHealthBarComponent.jsx

```diff
@@ -31,7 +31,8 @@ var AppHealthBarComponent = React.createClass({
 
     let allZeroWidthBefore = true;
     return health.map(function (d, i) {
-      var width = roundWorkaround(d.quantity * 100 / dataSum);
+      var width = (dataSum === 0) ? 0 : // Make sure we don't get NaN
+                  roundWorkaround(d.quantity * 100 / dataSum);
```

Comment:
Thanks for catching the lint comments, one which eslint isn't quite good enough to pick up is our ternary style preference:
```
var width = (dataSum === 0)
  ? 0 // Make sure we don't get NaN
  : roundWorkaround(d.quantity * 100 / dataSum);
```